src/sources/http_scrape/integration_tests.rs

```diff
@@ -0,0 +1,273 @@
+//! Integration tests for http_scrape source.
+//! The container configuration file is `docker-compose.http_scrape.yml`
+//! It leverages a static file server which serves the files in tests/data/http-scrape
```

Comment:
You might want to reference that this file server is the "dufs" referenced below. I had to double check that to make the connection.